### DISCUSSION TITLE
Replace tINIT with ftINIT in the guide

### DISCUSSION THREAD

**Discussion by johan-gson** [SysBioChalmers/Human-GEM#389] [General]
Are you guy's fine with that I replace tINIT with ftINIT in the Human-GEM guide? I have a small example when I run it on 60 samples from GTEx (takes ~2 hours to run). My plan was also to try to create a single-cell example that could explain bootstrapping etc. to estimate uncertainty, the same way I did in https://www.biorxiv.org/content/10.1101/2022.04.25.489379v1. I would like to get modeling from single-cell RNA-Seq started in the community this way.

**Comment by JonathanRob** [comment]
That's probably fine, but I wonder if it makes sense to keep the original tINIT instructions as well, since some may want to use the original algorithm? I'm open to it being replaced though.

**Reply by johan-gson** [reply]
Ok, I can keep the old version, but maybe I will refer to it as legacy? I'm thinking I will wait until both pull requests (The Human-GEM + RAVEN) are approved.
